opsdroid/connector/matrix/create_events.py

```diff
@@ -119,7 +119,7 @@ async def create_message(self, event, roomid):
         return events.Message(**kwargs)
 
     async def _file_kwargs(self, event, roomid):
-        url = self.connector.connection.get_download_url(event["content"]["url"])
+        url = event["content"]["url"]
```

Comment:
Am I correct in assuming this will be the `mxc://` url? This needs to be converted into a https URL, for people to be able to interact with it in a sensible way.

Reply:
Which is what that function used to do.

Reply:
I think you need https://github.com/poljar/matrix-nio/blob/master/nio/client/async_client.py#L768 :smile: